### DISCUSSION TITLE
thoughts regarding darts theorist package

### DISCUSSION THREAD

**Discussion by musslick** [AutoResearch/autora#372] [Ideas]
Here are some ideas/thoughts with regards to @hollandjg 's request to review the darts-theorist package.
Repo: https://github.com/AutoResearch/autora-theorist-darts
PyPI Package: https://pypi.org/project/autora-theorist-darts/
Docs: https://autoresearch.github.io/autora-theorist-darts/

1. the homepage link on pypi is dead; as Ben noted here https://github.com/AutoResearch/autora/discussions/366; it should be http://www.empiricalresearch.ai/ (not https)
2. I was wondering if we can make the "Quickstart" documentation part of the theorist template.
3. More generally, as we keep finding things to add to/remove from the template; I become increasingly convinced of @younesStrittmatter  suggestion here: #370 
4. I think it's fine that the weber-synthetic is not included here; it's better to use a general synthetic example
5. Once we have the synthetic examples as a separate package, we could add @benwandrew 's tutorial on the exponential learning curve.

Rest looks great!

**Comment by hollandjg** [comment]
Todos:
1. [x] Fix homepage link – now working at https://pypi.org/project/autora-theorist-darts/
2. [ ] Quick start documentation should be part of the theorist template: what do you think @younesStrittmatter ?
3. [x] single template is already on the way
4. [x] There's now an autora-synthetic-data package with the existing data, we'll integrate that for the documentation.
5. [ ] Ditto. The autora-synthetic-data needs reviewing and possibly a bit of restructuring, but here it is: https://github.com/AutoResearch/autora-synthetic-data
